contributors/devel/sig-storage/flexvolume.md

```diff
@@ -84,7 +84,7 @@ In addition to the user-specified options and [default JSON options](#default-js
 ```
 kubernetes.io/pod.name
 kubernetes.io/pod.namespace
-kubernetes.io/pod.uid
+kubernetes.io/pod.UID
```

Comment:
The lowercase looks correct:
https://github.com/kubernetes/kubernetes/blob/330b5a2b8dbd681811cb8235947557c99dd8e593/pkg/volume/flexvolume/driver-call.go#L59:2